public/app/features/storage/RootView.tsx

```diff
@@ -83,9 +83,8 @@ export function RootView({ root, onPathChange }: Props) {
         <InlineField grow>
           <FilterInput placeholder="Search Storage" value={searchQuery} onChange={setSearchQuery} />
         </InlineField>
-        <Button className="pull-right" onClick={() => onPathChange('', StorageView.AddRoot)}>
-          Add Root
-        </Button>
+        <div className="page-action-bar__spacer" />
```

Comment:
Would the `Space` component work here?

Reply:
don't think so cause it needs to flex. looks like `Space` only supports hardcoded values right now 🤔

Reply:
I think an empty Stack component could work then? If we want to get rid of this class as well.

Reply:
in this case it's already a global emotion style, so i'm not too fussed about getting rid of it immediately 🤷